### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by a1346054**

### CODE REVIEW

**Comment by m-o-e** [comment]
Thanks! 🌈

File: lib/lolcat/cat.rb [hunk 1]
```diff
@@ -51,8 +51,8 @@ def self.cat!
       opt :invert, "Invert fg and bg", :short => 'i', :default => false
       opt :truecolor, "24-bit (truecolor)", :short => 't', :default => false
       opt :force, "Force color even when stdout is not a tty", :short => 'f', :default => false
-      opt :version,  "Print version and exit", :short => 'v'
```
Comment (Legin1976) [inline lib/lolcat/cat.rb:54]: 4879 1700 4723 1999

**Review by Legin1976 — APPROVED** [review]

**Review by Moazzem-Hossain-pixel — COMMENTED** [review]
Please take steps to resolve.

File: lib/lolcat/cat.rb (hunk 1 above)
Comment (Moazzem-Hossain-pixel) [reply lib/lolcat/cat.rb:54]: If you can take the necessary initiate that will be highly appreciated.

**Pull request merged**